src/js/core/methods/EosGetPublicKey.js

```diff
@@ -9,24 +9,19 @@ import * as UI from '../../constants/ui';
 import { UiMessage } from '../../message/builder';
 import type { EosPublicKey } from '../../types/networks/eos';
 import type { CoreMessage, UiPromiseResponse } from '../../types';
-
-type Batch = {
-    path: Array<number>;
-    showOnTrezor: boolean;
-}
-
-type Params = Array<Batch>;
+import type { MessageType } from '../../types/trezor/protobuf';
 
 export default class EosGetPublicKey extends AbstractMethod {
-    params: Params;
+    params: $ElementType<MessageType, 'EosGetPublicKey'>[] = [];
     hasBundle: boolean;
-    confirmed: boolean = false;
+    confirmed: ?boolean;
```

Comment:
you will probably do global find and replace `confirmed: ?boolean;`

Reply:
yes, already done